org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CompletionHandlerTest.java

```diff
@@ -1178,6 +1180,99 @@ public void testCompletion_testClassesAvailableIntoTestCode() throws Exception {
 		assertEquals("AbstractTest - foo.bar", list.getItems().get(0).getLabel());
 	}
 
+	@Test
+	public void testCompletion_overwrite() throws Exception {
```

Comment:
default is now to not overwrite so this test really becomes testCompletion_insert